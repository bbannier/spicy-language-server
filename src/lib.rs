#![feature(str_strip)]

#[macro_use]
extern crate pest;

pub mod lsp;
pub mod options;
pub mod spicy;
