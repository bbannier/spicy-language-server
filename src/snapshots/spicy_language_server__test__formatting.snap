---
source: src/lib.rs
expression: "server.0.formatting(DocumentFormattingParams\n{\n    text_document: TextDocumentIdentifier::new(uri), options:\n    FormattingOptions::default(), work_done_progress_params:\n    WorkDoneProgressParams::default(),\n}).await"
---
Ok(
    Some(
        [
            TextEdit {
                range: Range {
                    start: Position {
                        line: 0,
                        character: 0,
                    },
                    end: Position {
                        line: 0,
                        character: 27,
                    },
                },
                new_text: "module foo;",
            },
        ],
    ),
)
